
\documentclass[USLetter,11pt]{article}
\usepackage{fleqn}
\usepackage{amssymb,amstext}
\usepackage{array}
\usepackage[latin1]{inputenc}
\usepackage{graphicx}
\usepackage{amsmath}
\usepackage[margin=1in]{geometry}
\usepackage{array}
\usepackage{amsmath}
\usepackage{latexsym}
\usepackage{psfrag}
\usepackage{graphicx}
\usepackage{caption2}
\usepackage{url}


\usepackage{pgf}
\usepackage{tikz}


\newcommand\bbone{\ensuremath{\mathbbm{1}}} 

\usepackage{natbib}

%\bibliographystyle{sysbio}
\bibliographystyle{rspublicnat}

\urlstyle{tt}


\renewcommand\captionmargin{0cm}
\renewcommand{\captionlabelfont}{\bf}
\renewcommand{\captionlabeldelim}{}
\captionstyle{centerlast}


\newcommand{\rep}[3][1]
{
\psfrag{#2}[c][c][#1]{#3}
}


\renewcommand{\baselinestretch}{2}


\begin{document}


\newcommand{\avg}[1]{\left< #1 \right>} % for average
\newcommand{\Ne}{N_\mathrm{e}}
\newcommand{\Rp}{\theta}
\newcommand{\R}{R}
\newcommand{\Rt}{\R_{t}}
\newcommand{\D}{D}
\newcommand{\Dt}{\D_{t}}
\newcommand{\V}{V}
\newcommand{\Vt}{\V_{t}}
\newcommand{\T}{T}
\newcommand{\Rmin}{{\R_{\infty}}}
\newcommand{\dd}{\mathrm{d}}


\newcommand{\Lr}{L}
\newcommand{\Lmin}{\Lr_{\infty}}
\newcommand{\Lmina}{\alpha}
\newcommand{\Lminb}{\beta}
\newcommand{\Lra}{\gamma}
\newcommand{\Lrb}{\delta}


\begin{tabular}{p{0.5\linewidth}>{\raggedleft}p{0.5\linewidth}}
& \vspace{-2.5cm} February 18, 2016 \\
\end{tabular}

\noindent

\noindent
{\bf \Large  The Red-Queen model of recombination hot-spot evolution: a theoretical investigation.} \\


\noindent
Thibault Latrille, Laurent Duret, Nicolas Lartillot

\noindent
\textit{\small Laboratoire de Biométrie et Biologie Evolutive, Lyon, France.}
\\
\texttt{nicolas.lartillot@univ-lyon1.fr}
\noindent
{\bf Running head:} The recombination Red-Queen

\vspace{1cm}

\noindent
{\bf Keywords:} 

\newpage

%%%%%%%%%%%%%%%%%%%%
%  ABSTRACT
%%%%%%%%%%%%%%%%%%%%

\section*{\sc Abstract}

In humans and many other species, recombination events cluster in narrow hotspots distributed across the genome, whose location is determined by the Zing-finger protein PRDM9. Hotspots are not shared between human and chimpanzee, suggesting that hotspots are short-lived. To explain this fast evolutionary dynamics of recombination landscapes, an intra-genomic Red-Queen model, based on the interplay between two antagonistic forces, has been proposed. On the one hand, biased gene conversion, mediated by double-strand breaks, results in a rapid extinction of hotspots in the population. On the other hand, the resulting genome-wide depletion of recombination induces strong positive selection favoring new Prdm9 alleles recognizing new sequence motifs across the genome and restoring normal levels of recombination. Thus far, however, this Red-Queen scenario has not been formalized as a quantitative model.

Here, we propose a population-genetic model of the Red-Queen dynamic of recombination. This model was implemented as a Wright-Fisher simulator, allowing exploration of the behaviour of the model (in terms of the implied mean equilibrium recombination rate, diversity at the PRDM9 locus, or turnover rate) as a function of the parameters (effective population size, mutation rate). In a second step, analytical results, based on self-consistent mean-field approximations, were derived. These analytical results reproduce the scaling relations observed in the simulations, offering key insights about the detailed population-genetic mechanisms of the Red-Queen model. These insights and scaling relations can now be tested against empirical data currently available in mammals.

\newpage

\section*{Introduction}

In mammals, as in many other species, meiotic recombination events are not uniformly distributed along chromosomes. Instead, they tend to occurs within narrow regions, typically of length 1-2 kb, called hotspots (Jeffreys 2001, Myers, Mancera). About 33 000 hot spots have been identified in humans, a similar number in chimpanzees, and ... in the mouse. Surprisingly, hot spots are not conserved between humans and chimps (refs), suggesting that recombination landscapes are highly dynamic, undergoing a rapid turnover within relatively short evolutionary times. The broad-scale distribution of recombination (at the megabase scale), on the other hand, is conserved between human and chimpanzees, and more generally on a broader evolutionary scale, primarily reflecting the karyotypic structure, in particular, with higher recombination rates in sub-telomeric regions, or on smaller chromosomes.

The location of hot spots is primarily determined by the zn finger protein PRDM9. Upon binding DNA at specific sequence motifs of approximately 13 bp through its ZnFinger domain (Baudat et al. 2010, 2013; Myers et al. 2010), PRDM9 catalyzes histone H3 Lys4 trimethylation (H3K4me3) at neighboring nucleosomes through its SET domain. The local changes in chromatin structure lead to the recruitment of Spo11, triggering the formation of a double strand break (DSB) in the immediate vicinity of the target site. This DSB is repaired, ultimately leading to a crossover (CO) or a non-crossover (NCO) recombination event. In both cases, the DNA sequence around (and including) the binding site of PRDM9 is resected from the chromosome that has undergone the DSB and is repaired by using the homologous chromosome as a template over approximately 100 to 1000 base pairs ? a process called gene conversion.

When the two chromosomes differ due to the presence of heterozygous sites (polymorphisms), an event of gene conversion leads to a loss of information (loss of the allele carried by the broken chromosome). As a result, gene conversion has the following paradoxical consequence: if both paternal and maternal chromosomes carry the wild type copy of the 13 bp PRDM9 binding site, each of them is equally likely to be targeted by PRDM9, to undergo the DSB and thus to be converted by its homologue. If, on the other hand, one of the chromosomes has a mutant copy of the binding site that partially or completely inactivates the binding of PRDM9, then PRDM9 will preferentially target the other chromosome ? whose wild-type binding site will then be converted into the mutant copy present on the homologue, thus resulting in the loss of the functional allele. This process of DSB-induced Biased Gene Conversion (biased in favor of mutant hotspots) will hereafter be referred to as dBGC. At the level of the population, dBGC results in a progressive accumulation of inactive hotspot mutant alleles by mutation and preferential transmission of the mutants, leading to an increase in frequency, and the ultimate fixation, of inactive mutant hotspots genome-wide. Hence, through the action of dBGC, recombination hotspots turn out to be self-destructive, a phenomenon referred to as the hotspot paradox (Boulton et al. 1997).

Population genetics arguments suggest that dBGC might be a sufficient force to lead to the rapid extinction of recombination hotspots over the genome (Coop and Myers 2007, Ubeda and Wilkins 2011, Lesecque...), thus raising the question of how recombination is maintained in the long run. A remarkable observation in this respect is that PRDM9 is the most rapidly evolving gene in primates and rodents, and more generally across metazoans, and is highly polymorphic, specifically in its DNA binding domain (Buard, Ponting 2011). There is clear evidence that this ZnFn domain is subject to strong positive selective pressure to change of DNA target (Ponting 2011), which ultimately results in the targeting of new hotspots at other locations. Thus, positive selection at the PRDM9 locus, leading to the recruitment of new hotspots genome wide, would seem to represent a good candidate for the mechanism compensating the loss of hotspots by dBGC.

Altogether, this interplay between dBGC, leading to the loss of current hotspots, and positive selection on PRDM9, leading to the recruitment of new hotspots, appears to provide a convincing model for explaining the rapid evolutionary turnover of recombination landscapes and currently represents the dominant hypothesis in the literature (Myers et al. 2010, Ubeda and Wilkins 2011, Lesecque et al. 2014). Interestingly, it represents an instance of intra-genomic conflict, where two processes are chasing after one another. As such, it is analogous to Red Queen situations, such as arms races between hosts and pathogens, often encountered in evolutionary ecology. According to this Red Queen model of recombination turnover, PRDM9 would have to evolve constantly to compensate for the loss of its targets caused by dBGC and maintain a sufficient number of recombination hotspots in the genome.

Several observations are consistent with the predictions of the Red Queen model of recombination landscape dynamics. First, the analysis of transmission segregation by sperm-typing at a few loci demonstrated that some recombination hotspots are subject to dBGC in humans and mice (Jeffreys and Neumann 2002, 2009; Baker 2015). Second, comparative genome analyses have suggested that PRDM9 target motifs accumulate mutations at an accelerated rate genome wide (Myers 2010, Lesecque 2014, Baker 2015, Davies). Given that recombination is required for the proper segregation of chromosomes, and that Prdm9 knockout mice are sterile (Brick et al. 2012), it seems plausible that the loss of recombination hotspots might affect fertility, thus inducing positive selection on PRDM9 for eliciting new sequence motifs. 

In many respects, however, the Red Queen model is still speculative and needs to be further theoretically investigated and empirically tested. A first specific issue is how exactly positive selection is leveraged onto PRDM9 by hotspot extinction. At least two alternative hypotheses can be examined: (1) the genome-wide depletion model: global depletion of hotspots across the entire genome could lead to improper chromosome segregation and/or to the build-up of linkage disequilibrium, resulting in increased levels of background selection and hindering adaptation; (2) the key hotspot model: the loss of fitness might be caused by the loss of key hotspots at some specific loci where recombination is essential (Ponting 2011). For instance, meiosis in males requires one crossover event in the pseudoautosomal region of sex chromosomes. Alternatively, specific loci (e.g. the MHC region) are known to be under strong positive selection, thus requiring high local rates of recombination. A lack of recombination hotspot in such specific regions of the genome might be sufficient to trigger a selective pressure on PRDM9. Which of the two models more adequately describes true empirical situations is still a totally open question, amenable to empirical investigation.

Second, a more accurate picture of the Red Queen model should really integrate the role of genetic diversity. Thus far, the Red Queen has been stated mostly in terms of a succession -- global hotspot extinction by dBGC followed by allelic replacement at the PRDM9 locus. However, in reality, the PRDM9 locus is known to be highly polymorphic, with many alternative alleles, each targeting a different subset of hotspots. How to explain this genetic diversity, and how does the Red Queen model unfolds once this polymorphism is fully accounted for? Intuitively, positive selection for new PRDM9 alleles starts to be active before old alleles have lost all of their hotspots. In turn, emergence of these new alleles is expected to distribute recombination rates more evenly across a larger number of weaker hotspots genome-wide, thereby weakening the effect of dBGC and thus slowing down extinction of old hotspots. Thus, a more accurate description of the Red Queen dynamics should be formulated in terms of a complex mix of dBGC, selection and loss of PRDM9 alleles, either by random drift or by progressive obsolescence through extinction of their targets. In this context, demography is expected to play an important role, since larger populations should result in more alleles segregating at the PRDM9 locus. %These various factors are expected to create complex but interesting mutual dependencies between effective population size, diversity at the PRDM9 locus and the evolutionary dynamics of recombination landscapes (variance and rate of turnover) ? all of which are entirely unexplored, yet are amenable to theoretical investigation.

A question of particular interest is how the stationary regime of the Red Queen, in terms of the mean levels of depletion of recombination rates, the equilibrium diversity at the PRDM9 locus and the rate of turnover of recombination landscapes, scales with population size and with the parameters of the genetic system (in particular, the mutation rates at the target sites and at the minisatellite locus encoding the Zn-finger domain of PRDM9). Based on current empirical evidence, both dBGC and PRDM9 are likely to be implicated in the evolutionary dynamics of recombination landscapes across placental mammals, with the notable exception of the dog lineage. On a broader scale, PRDM9 is present across the metazoan tree of life, although absent in birds and in some other specific lineages (refs). Across mammals, and even more critically across metazoans, both mutation rates and effective population size are likely to vary over 1 to 2 orders of magnitude. Thus, if the Red Queen is to provide a model of the evolutionary dynamics of recombination, not just in humans, but more globally across mammals or metazoans, it is fundamental to better understand how the working regime of this model effectively scales as a function of its parameters, and whether this scaling is such that the Red Queen indeed provides an empirically reasonable model over the empirically relevant range of variation.

With this in mind, here, we introduce a simple population-genetic model of the Red Queen of recombination. The model was implemented as a simulation program and run over a broad range of conditions, so as to explore its qualitative behavior, as well as its scaling, as a function of the parameters. These simulation experiments were then backed up by analytical approximations, based on a self-consistent mean-field argument, which are meant to capture the main properties of the stationary regime of the process. Based on this analysis, we provide a general overview of the behavior of the Red Queen model in different parameter regimes. Finally, a empirical calibration of the model against currently available data in the mouse is attempted, which suggests both a high mutation rate at PRDM9 and strong dBGC on its targets.

\section*{Materials and methods}

\subsection*{Population-genetic model}

The evolutionary dynamics of the Red Queen was formalized as a Wright-Fisher model with mutation and selection. The population is assumed to be panmictic, with constant size $N_e$ and with non-overlapping generations. Only the genetic composition of the PRMD9 locus was explicitly modeled, the evolutionary dynamics at other loci (PRDM9 targets across the genome) being implicit.

{\bf Mutation.}
The locus PRDM9 mutates at constant rate $u$ per generation per locus. Each mutation produces a new functional PRDM9 variant, endowed with an entirely new set of $M$ targets, all of which are assumed to recombine at the same rate $r_0$. The number of targets is the same ($M$) for each allele, and there is no overlap between the targets of distinct PRDM9 alleles. Thus $u$ should be understood as a functional mutation rate (discounting loss-of-function mutants). At each generation, $K_{t}$ denotes the number of PRDM9 alleles in the population, and $\forall i \in \{ 1, \, \dots, \, K_{t} \}$, $n_{i,t}$ is the number of copies of the $i^{th}$ allele in the population. Consequently, $x_{i,t} = n_{i,t} / 2 \Ne$ is the frequency of allele $i$ at time $t$.

{\bf Recombination and erosion.}  
The recombination activity induced by an allele is maximal at the birth of this allele.  As time proceeds forward, however, the targets of the allele are progressively eroded by dBGC. This erosion is modelled implicitly, by tracking over time the fraction of active targets associated with each allele. This fraction is denoted as $\theta_{i,t}$ for allele $i$ at time $t$. In the following, this fraction will be called the recombination \emph{activity} induced by allele $i$. Because of dBGC, this fraction is a decreasing function through time.

We assume an additive model for the genome-wide recombination induced by a given genotype.
Specifically, consider a diploid individual, with alleles $i$ and $j$ and the PRDM9 locus. At time $t$, the activity of these two alleles are $\theta_{i,t}$ and $\theta_{j,t}$. The genome-wide recombination activity in this individual is assumed to be equal to:
\begin{eqnarray*}
R_{ij} &=& M r_0 \frac{\theta_{i,t} + \theta_{j,t}}{2}
\end{eqnarray*}
%This model can be justified by assuming that the PRDM9 protein is in limiting concentration in the cell, and that a fixed fraction of sites that are bound by the $PRDM9$ protein across the genome eventually initiate recombination. 
Note that this is assumed to be valid also for a homozygous individual (i.e. when $i=j$).

At any given time, most hotspots associated with a given PRDM9 allele are either fully active, or fully inactive (i.e. a minor fraction of hotspots are in polymorphic state). As a result, the rate at which the activity induced by allele $i$ decays is just the rate of substitution from active to inactive hot-spots at the level of the population. This rate is itself equal to the rate of inactivating mutations per target at the level of the population, $2 N_e v$, multiplied by the fixation probability of the inactive mutant. Under strong dBGC, the fixation probability is equal to $2r$, where $r$ is the recombination rate at the targets at the level of the population. This recombination rate is itself directly proportional to the frequency of the allele: $r = r_0 x_{i,t}$. 
Altogether, the activity induced by allele $i$ decays as:
\begin{eqnarray*}
\frac{d \theta_{i,j}} {d t} &=& - 2 N_e v \, 2 r_0 x_{i,t} \, \theta_{i,t}
\\
&=& -\rho \, x_{i,t} \,  \theta_{i,t},
\end{eqnarray*}
where we define $\rho = 4 N_e v r _0$.

{\bf Selection.} The fitness of an invidual is assumed to be an increasing function $f$ of its recombination activity $R$. For convenience, the function $f$ is parameterized in terms of the relative genome-wide recombination activity, i.e. $\theta = R / M r_0$, which is between 0 and 1. In the following, we will more specifically consider a power-law function:
\begin{eqnarray*}
f (\theta) \propto \theta^\alpha,
\end{eqnarray*}
where $\alpha$ is a parameter of the model. Large values of $\alpha$ induce a stronger selection against low recombination rates.
The fitness of an individual with genotype $(i,j)$ is thus:
\begin{eqnarray*}
\omega_{i,j} &=& f \left( \frac{\theta_{i,t} + \theta_{j,t}}{2} \right).
\end{eqnarray*}
The average fitness induced by allele $i$ over the population is then given by:
\begin{eqnarray*}
\overline{\omega_{i,t}} &=& \sum_{j=1}^{K_t}  \, x_{j,t} \, f \left( \dfrac{\Rp _{i,t} + \Rp _{j,t}}{2} \right)
\end{eqnarray*}
Finally, the mean fitness over the population is:
\begin{eqnarray*}
\overline \omega &=& \sum_{i=1}^{K_t} \, x_{i,t} \,  \omega_{i,t}
\end{eqnarray*}

{\bf Overall simulation cycle}. For each new generation, the simulation is decomposed in three steps, performed in the following order:
\begin{itemize}
\item
Mutation and creation of new alleles of PRDM9: the number of new alleles is drawn from a Poisson distribution with mean $2 \Ne u$. Each new allele is introduced at a frequency $1 / 2 \Ne$. It replaces a randomly chosen resident allele;
\item 
Erosion: the activity of each allele currently segregating in the population is eroded by a factor proportional to the  frequency of the allele:
$\Rp _{i,t+1} =  \Rp _{i,t} \operatorname{exp} \left( - \rho x_{i,t} \right)$, for $i=1..K_t$;
\item
Drift and selection: the new generation of $2 \Ne$ haploid copies is drawn from a multinomial distribution. The probability of drawing allele $i$ is equal to it's frequency $x_{i,t}$ time it's relative fitness $\omega_{i,t} / \bar \omega$.
\end{itemize}
The model was implemented in Python, the code is hosted at \url{https://github.com/ThibaultLatrille/RedQueen}.
Each simulation starts by a burn-in, and the the data a recorded only when the steady-state is reached. The burn-in phase is finished when all initial alleles in the population are extinct and replaced by new alleles. The number of generations simulated at the steady state is $50$ times the number of generations necessary during burn-in. \\

\subsection*{Summary statistics} 

To explore the behaviour of the Wright-Fisher model as a function of the parameters, four summary statistics were considered. These statistics are meant to capture key features of the Red-Queen dynamics: the diversity at PRDM9 locus, the mean recombination rate over the population at stationarity, a measure of the variance of recombination landscapes, and the time of turnover of the genetic diversity at the PRDM9 locus (or, equivalently, the time of turnover of recombination landscapes). These summary statistics are computed at stationarity and are averaged over long simulation trajectories. 

{\bf Diversity of PRDM9.} Diversity can be defined in several ways; for example, one could consider the number of distinct alleles $K_{t}$. One shortcoming of this definition is that $K_t$ is strongly sensitive to the behavior of low-frequency PRDM9 alleles, most of which never invade the population and therefore do not meaningfully contribute to the macroscopic behavior of the model.
An alternative, more relevant, measure of diversity is the inverse of the homozygosity: 
\begin{eqnarray*}
     \D &=& \avg{\Dt} = \avg{\dfrac{1}{\sum_{i \in K_{t}} x_{i,t}^2}} 
\end{eqnarray*}
$D^{-1}$ is also known as Simpson's index \cite{Hill1973}.
%It obeys to the replication principle, which states that if we have N equally large, equally diverse groups of PRDM9 alleles with no alleles in common, the diversity of the pooled groups must be N times the diversity of a single group.
Note that $D$ equals $K$ if all alleles have equal weights and is close to $1$ if one allele dominates the population. Thus, $D$ can be seen as the \emph{effective} number of PRDM9 alleles.

{\bf Mean relative recombination rate.} Since the model linking the activity of each alleles (the fraction of active targets) and genome-wide recombination is additive, the average relative genome-wide recombination rate at the population level is equal to:
\begin{eqnarray*}
    \R &=& \avg{\Rt} =\avg{\sum_{i \in K_{t}} x_{i,t} \Rp _{i,t}} 
\end{eqnarray*}
The absolute recombination rate is simply $M r_0$ times the relative recombination rate.

{\bf Relative variance of recombination landscapes.}
In the limit where the number of hotspots in the genome is small compared to genome size, the relative variance in recombination rates in the genome is given by 
\begin{eqnarray*}
     \V &=& \avg{\Vt} = \avg{\sum_{i \in K_{t}} x_{i,t}^2 \Rp _{i,t}} 
\end{eqnarray*}
Again, a scaling factor of $M^2 r_0^2$ has been discounted here.

{\bf Turn-over time.}
The turn-over time is defined as the decorrelation time of the diversity at the PRDM9 locus. This can be quantitatively assessed using the \emph{cross-homozygosity}, which is defined as the fraction of homozygotes in a population that would be obtained by hybrdizing populations at time $t$ and $t+T$ in equal proportions. The cross-homozygosity reduces to the regular (or instant) homozygosity for $T=0$, and it drops to 0 for large $T$. The turnover time is defined as the time $T$ for which the cross-heterozygosity is equal to half of the instant homozygosity.
\begin{eqnarray*}
    \T = \avg{\dfrac{\sum_{i \in K_{t}} x_{i,t} x_{i,t+T}}{\sum_{i \in K_{t}} x_{i,t} x_{i,t}}  }  &=&  \dfrac{1}{2}.
\end{eqnarray*}

\subsection*{}

A series of phase diagrams, showing the scaling relations between summary statistics and parameters of the model, were obtained as follows. For each parameter, $64$ independent simulations were run, each time changing the value of only one parameter around it's central value. The range around the central value of the parameter is $10^{4}$, meaning the minimum value (the left of the diagram) is $10^2$ times lower than the central value, and maximum value (the right of the diagram) is $10^2$ times greater than the central value.\\
The mean and the variance of each summary statistic was calculated for each simulation, and plotted as a function of the parameter.
The central value for all phase diagrams are :
\begin{itemize}
\item $\Ne=10^{5}$ : The population size
\item $u=10^{-6}$ : The mutation rate at the PRDM9 locus
\item $v=10^{-6}$ : The mutation rate at the hotspots locus
\item $r_0=10^{-3}$ : The recombination rate of active targets
\item $\alpha=10^{-1}$ : The exponent of the fitness function
\end{itemize}

\newpage

\section*{Results}
\subsection*{Simulation results} 

A typical trajectory of the simulation is displayed in figure 1. As time proceeds forward, new PRDM9 alleles, created by mutation, invade the population, increase in frequency, reaching a peak and then decreasing until being lost and replaced by new alleles (top panel). Tracking the activity of each allele through time (bottom panel) shows that new alleles start with maximal recombination activity. Progressively, however, their targets are eroded by dBGC. When the activity induced by an allele goes below the average activity over the population (whose equilibrium value over time, $R$, is shown as a straight line in the bottom panel), this allele starts to be selected against, decreasing in frequency, until being eliminated from the population. Of note, the activity of a dying allele is typically not 0. Instead, it reaches a finite level, whose average value ($R_\infty$) will be considered further below.

As a way to capture the key features of the Red-Queen dynamics at stationarity, four summary statistics were considered: mean recombination activity, genetic diversity at the PRDM9 locus, relative variance of recombination landscapes, and turnover time (i.e. the decorrelation time of genetic composition at the PRDM9 locus). For a given configuration of parameters of the model (effective population size $N_e$, mutation rate at the PRDM9 locus $u$, mutation rate at the targets $v$, strength of selection on recombination $\alpha$), these four statistics were averaged over long simulation trajectories. Their scaling behavior as a function of the four main parameters are displayed on figure 2, 3, 4, and 5. Several points are noteworthy.

First, the equilibrium recombination rate is suboptimal: $\R < 1$ (figure 2). Equivalently there is a recombination \emph{gap} caused by the Red-Queen. This gap induces a recombination \emph{load}, which can be defined as the relative decrease in fitness between the optimum recombination activity $\R=1$ and the realized recombination activity at equilibrium $\R < 1$. One main question of interest, which will be further considered below, is to characterize and quantify this recombination load. Another important observation, the equilibrium recombination rate ($\R$) is constant as a function of $\Ne$. It increases with $u$ and $\alpha$ and decreases with $v$.

Second, plotting the PRDM9 diversity ($\D$) as a function of $\Ne$ and $u$ (figure 3) reveals two distinct regimes. The transition between the two regimes appears to essentially depend on the scaled mutation rate ($\Ne u$) at the PRDM9 locus. For low scaled mutation rates ($N_e u  << 1$), at any given time, a single PRDM9 allele dominates the population ($D=1$). In this \emph{succession} regime, PRDM9 alleles replace each other through hard sweeps -- which corresponds to the typical verbal description of the Red Queen dynamics. In this regime, increasing $N_e$ merely accelerates the dynamics, without changing anything else to the overall morphology of the process. In particular, at any time, recombination is concentrated on one single set of recombination hotspots (corresponding to the currently dominating PRDM9 allele), and thus the variance of recombination landscapes is maximal.

On the other hand, when $\Ne u$ is sufficiently high, successive waves of erosion-invasion start to overlap, such that multiple PRDM9 alleles now coexist in the population. In this polymorphic regime, diversity at the PRDM9 is roughly proportional to $\Ne u$. As a consequence, in this regime, the consequence of an increase in $N_e$ is not anymore an acceleration of the dynamics, as in the succesion regime. Instead, when $N_e$ increases, PRDM9 diversty increases, such that recombination spreads over a larger number of more weakly-recombining hot-spots, thus resulting in an overall decrease in the variance of recombination landscapes. Conversely,  the fact that the recombination hot-spots become weaker exactly compensates for the  increase in the strength of biased-gene conversion caused by the larger effective population size, and thus hot-spot lifetime (turn-over time) now remains constant as a function of $\Ne$.


\subsection*{Analytical approximations} 

In the following, we derive analytical approximations of the Red Queen model introduced above. These approximations are meant to capture the scaling relations between summary statistics and model parameters observed in the simulation experiments. They will also provide mechanistic insights into the inner working of the Red Queen process.

The two regimes, succession and polymorphic, are considered in turn. In both cases, the general structure of the argument is as follows. In principle, the changes in frequency of a typical PRDM9 allele are determined by a combination of selective effects and random drift. Furthermore, selection normally depends on the relative difference between the fitness of the allele and the constantly fluctuating fitness background of the population. Our main simplifications are to (1) ignore stochastic effects and to consider a deterministic version of the model (strong selection); (2) to consider that the mean fitness of the population is constant through time, being equal to its time average (mean-field approximation); (3) to assume that the differences in fitness between alleles are small, so that we can linearize the fitness function as a function of the $\theta_i$'s (linear model).

Linearizing the model allows us to express the mean fitness of the population, and therefore also the selection coefficient experienced by a new allele entering the population, as a function of $\R$, the mean recombination activity in the population. This selection coefficient determines the mean waiting time $\tau$ between successive invasions of the population by new alleles. Thus, we can express $\tau$ as a function of $\R$ and the parameters of the model. Conversely, knowing the waiting time before the next invasion, and accounting for ongoing erosion of recombination for resident alleles over this time, leads to an estimate of the mean equilibrium recombination activity of the population $\R$, as a function of $\tau$. Combining these two relations and eliminating $\tau$ leads to a self-consistent mean-field estimate of $\R$. From there, we can derive  estimates for all other summary statistics.

The self-consistent estimate of $\R$ obtained using this approach is implicit: it is the solution of an equation of the form $R = g(R)$, which can be solved only numerically. An explicit approximation will be derived under the additional assumption that the recombination gap is small $1 - \R << 1$ (small-load approximation).

\subsection*{Analytical approximations in succession regime} 

In the succession regime, there is essentially one PRDM9 allele at a time, whose frequency is close to 1. As a consequence, the recombination activity associated with this allele decreases at constant rate $\rho = 4 \Ne v r_0$:
\begin{eqnarray*} 
   \dfrac{\dd \Rp_{t}}{\dd t}  &=&  - \rho \Rp_{t},
\end{eqnarray*}
and $\Rp_{t}$ follows an exponential decrease through time:
\begin{eqnarray*} 
   \Rp_{t} &=&  e^{- \rho t}.
\end{eqnarray*}
Let us call $\tau$ the mean time between two successive invasions. We now derive two independent relations between $\tau$ and $\R$.

First, we can derive an approximation for the rate of invasion of the population by a new PRDM9 allele using a simple population genetic argument. This rate is equal to the rate of mutation at the PRDM9 locus at the level of the population, $2 N_e u$, multiplied by the probability of invasion, which is itself equal to $ 2 s$, where $s$ is the selection coefficient experienced by the new allele. This coefficient depends on the current activity of the resident allele, $\theta_t$ (the activity of the invading allele is 1). After linearization, it can be expressed as:
\begin{eqnarray*} 
s &=& \dfrac{f'(\Rp_{t})}{f(\Rp_{t} )}  \dfrac{(1 - \Rp_{t} )}{2}.
\end{eqnarray*}
In principle, this selection coefficient depends on time, through $\theta_t$. However, we can approximate it by a constant coefficient, by just replacing $\theta_t$ by the average activity $\R$ into this equation;
\begin{eqnarray*} 
s &\simeq& \dfrac{f'(\R)}{f(\R)}  \dfrac{(1 - \R )}{2}.
\end{eqnarray*}
This leads to the following estimate for the rate of invasion:
\begin{eqnarray*} 
\lambda &\simeq&
\mu \dfrac{f'(\R)}{f(\R)} \dfrac{(1 - \R )}{2},
\end{eqnarray*}
where $\mu = 4 \Ne u $ is the scaled mutation rate at the PRDM9 locus.
The inverse of $\lambda$ gives an approximation to $\tau$, thus leading to our first relation:
\begin{eqnarray} 
\label{tausuccession}
\tau &\simeq&
\dfrac{1}{\mu} \dfrac{f(\R)}{f'(\R)} \dfrac{2}{(1 - \R)}.
\end{eqnarray}

Second, the mean recombination activity is simply the time average of $\theta_t$ over the time where the allele dominates the population. This time is (on average) equal to $\tau$, thus leading to the following approximation:
\begin{eqnarray} 
\label{Rsuccession}
\R &\simeq& \frac{1}{\tau} \,  \int_0^\tau \, e^{-\rho t} \dd t = \frac{1 - e^{-\rho \tau} }{\rho \tau}.
\end{eqnarray}
Noting that the minimum activity of a dying allele, at the time where it is being replaced by a new allele (i.e. at time $\tau$), is equal to:
\begin{eqnarray} 
\label{Rminsuccession}
\Rmin &=& \operatorname{e}^{-\rho \tau},
\end{eqnarray}
equation \ref{Rsuccession} can be rewritten as:
\begin{eqnarray*} 
1-\Rmin  + \R  \operatorname{log}(\Rmin )  &=& 0.
\end{eqnarray*}
This relation between $\R$ and $\Rmin$ will also be found in the polymorphic regime.

Combining these two relations, i.e. equations \ref{tausuccession} and \ref{Rsuccession}, leads to a self-consistent equation on $\R$:
\begin{eqnarray*} 
\R &=& \frac{1 - e^{-\dfrac{\rho}{\mu} \dfrac{f(\R)}{f'(\R)} \dfrac{2}{(1 - \R)}}} {\dfrac{\rho}{\mu} \dfrac{f(\R)}{f'(\R)} \dfrac{2}{(1 - \R)}}.
\end{eqnarray*}
which can be re-expressed more compactly as:
\begin{eqnarray}
\label{selfconsistent} 
R &=& g \left( \R, \dfrac{\rho}{\mu}\right).
\end{eqnarray}
For fixed $\dfrac{\rho}{\mu}$, $g$ is a strictly monotonous function (figure 6, in the case of a power-law fitness function), and has thus a unique solution. This solution can be obtained numerically, giving an estimate for $\R$.

Note that this self-consistent relation on $\R$ implies that $\R$ is independent of $\Ne$, as our simulation was suggesting. More specifically, $\R$ depends on the parameters of the model only through the compound parameter $\dfrac{\rho}{\mu} = \dfrac{v r_0}{ u}$.

Once a numerical estimate of $\R$ is available, all other quantities of interest can be expressed as a function of $R$ and the parameters of the model. The diversity at the PRDM9 locus $\D$ is simply equal to $1$ since there is only one allele. The relative variance of recombination landscapes is maximal, i.e. $\V = \R$, since recombination is concentrated on the targets of a single PRDM9 allele. Finally, the turn-over time $\T$ is simply equal to the mean time between two successive invasions $\tau$, already given above as a function of $\R$ (equation \ref{tausuccession}). Note that this time is inversely proportional to $N_e$. In other words, in the succession regime, changes in $N_e$ merely change the time scale of the red queen dynamics, without any effect on the stationary state.

\subsection*{Analytical approximations in polymorphic regime}

Ignoring random drift and linearizing the fitness as a function of the activities, a close set of differential equations for the frequencies of PRDM9 alleles ($x_{i,t}$) and their associated recombination rate ($\Rp_{i,t}$) can be derived: 
\begin{equation}
\left\{
      \begin{aligned}
          \dfrac{\dd x_{i,t}}{\dd t} &= \dfrac{f'(\Rt )}{2 f(\Rt )} \left( \Rp_{i,t}  - \Rt  \right) x_{i,t}, \;
             \forall i \in \{ 1, \, \dots, \, K_{t} \} \\
        \dfrac{\dd \Rp_{i,t}}{\dd t} &= 
        - \rho x_{i,t} \Rp_{i,t}, \;
             \forall i \in \{ 1, \, \dots, \, K_{t} \} \\
             \Rt &= \sum_{i \in K_{t}} x_{i,t} \Rp _{i,t} 
      \end{aligned}
\right. 
\end{equation}

Under the assumption of many alleles co-segregating in the population, $\Rt=\R$ is essentially constant, owing to an averaging effect over the specific trajectories of each allele. This leads to a decoupling of the above system of equations. As a result, we can focus on the trajectory of a single typical allele, with frequency $x_{t}$ and relative recombination rate $\Rp _{t}$:
\begin{equation}
\left\{
      \begin{aligned}
          \dfrac{\dd x_{t}}{\dd t} &= \dfrac{f'(\R )}{2 f(\R )} \left( \Rp_{t}  - \R \right) x_{t} \\
        \dfrac{\dd \Rp_{t}}{\dd t} &= 
        - \rho x_{t} \Rp_{t} \\
      \end{aligned}
\right.
\end{equation}
Here, $\R$ is now an external parameter, which will be determined in a second step, using a self-consistent argument.

A numerical solution of this system of equations is shown in figure 7: starting from a low frequency and a maximal activity, a typical allele first increases in frequency. Concomitantly, its activity is progressively eroded, at an increasingly higher rate. When the activity reaches the (now externally given) $\R$, the allele frequency reaches its maximum, after which it starts to decrease, ultimately going to 0. Meanwhile, the activity converges to a strictly positive asymptotically value $\Rmin$.

This system of equations is not analytically solvable as a function of $t$.
On the other hand, $\Rp_{t}$ is monotonic, and $x$ can be analytically expressed as a function of $\Rp_{t}$
\begin{equation}
x(\Rp_{t} ) =\dfrac{f'(\R)}{2 \rho f(\R )} (1- \Rp_{t}  + \R  \operatorname{log}(\Rp_{t} )) + x_{\mathrm{initial}}
\end{equation}
%[would be nice to show the graph where we see this function together with the bundle of realized trajectories for a simulation]

Letting $t \to \infty$, $\theta_t$ converges to $\Rmin$ and $x(\theta_t)$ to 0. In addition, since $x_{\mathrm{initial}} = 1 / N_e$ is small, we can let it go to 0, which yields a relation between $\R$ and $\Rmin$:
\begin{eqnarray}
\label{polyRmin}
0 &\simeq  1-\Rmin  + \R  \operatorname{log}(\Rmin ) 
\end{eqnarray}
Note that this is the same equation as in succession regime.

So far, we have considered $\R$ as an external parameter.
Now, if all alleles have the same trajectory but different arrival times, they all contribute back to $\R$. Using this argument, we can get a self-consistent estimation for  $\R$. This can be done by deriving two independent relations between $\tau$, the time between successive invasions, and $\R$ and the other parameters of the model.

First, using a tiling argument (see figure 8), we note that:
\begin{eqnarray*}
\int_0^{+\infty} \, x_t \dd t  &\simeq& \tau \sum_i x_i = \tau.
\end{eqnarray*}
Although $x_t$ is not analytically solvable as a function of $t$, the integral can be analytically computed (see appendix), leading to the following relation:
\begin{eqnarray*}
\tau &=& \dfrac{1 - \Rmin }{\rho \R } = - \dfrac{\operatorname{log}(\Rmin )}{\rho}.
\end{eqnarray*}
Inverting this equation gives:
\begin{eqnarray*}
\Rmin &=& \operatorname{e}^{-\rho \tau},
\end{eqnarray*}
which is the same expression for $\Rmin$ as that found in the succession regime.
Replacing $\Rmin$ by this expression in equation \ref{polyRmin} then gives:
\begin{eqnarray} 
\label{Rpoly}
\R &=& \frac{1 - e^{-\rho \tau} }{\rho \tau},
\end{eqnarray}
again, as in the succession regime.

Second, relying on the population-genetic argument used in the context of the succession regime, we can identify the time between two successive invasions $\tau$ as the inverse of the rate of invasion, which is equal to the mutation rate multipled by $2s$, where $s$ is the selection coefficient experienced by the newly invading allele, in a background recombination activity equal to $\R$:
\begin{eqnarray} 
\label{taupoly}
\tau &=&
\dfrac{1}{\mu} \dfrac{f(\R)}{f'(\R)} \dfrac{2}{(1 - \R)}.
\end{eqnarray}
Although derived using different arguments, these two relations, equations \ref{Rpoly} and \ref{taupoly}, are the same as those obtained under the succession regime (equations \ref{Rsuccession} and \ref{tausuccession}). Combining them together therefore leads to the same self-consistent equation for $\R$ (equation \ref{selfconsistent}), which gives us a numerical estimate of $\R$ also valid in the polymorphic regime.

Using the same tiling argument as above, we can get a numerical estimate of the diversity at the PRDM9 locus $\D$. Specifically:
\begin{eqnarray*}
\sum_i x_i^2 &\simeq& \dfrac{1}{\tau}  \, \int_{0}^{\infty } x_{t}^2 \dd  t,
\end{eqnarray*}
which is also analytically available, yielding:
\begin{eqnarray*}
\D = \left( \sum_i x_i^2 \right)^{-1} &=& \dfrac{4\rho f(\R )}{f'(\R )\left[ 1 + \Rmin  - 2 \R   \right]} 
\end{eqnarray*}
Similarly, we get the relative variance of recombination landscapes $\V$
\begin{eqnarray*}
\V &=& \dfrac{1}{\tau} \, \int_{0}^{\infty } x_{t}^2 \Rp_{t} \dd  t \\
&=& \dfrac{f'(\R )}{4 \rho f(\R )}\R \left[ 1 + \Rmin  - 2 \R   \right] 
\end{eqnarray*}
Finally, the turn-over time $\T$ can be determined by observing that, if $D$ is to be interpreted as the effective number of alleles at the PRDM9 locus, then the genetic diversity at this locus will be entirely renewed after $D$ successive invasions, each of which takes on average a time $\tau$. Therefore, $T = D \tau$, with $D$ and $\tau$  given above as a function of $R$, $\Rmin$ and the parameters of the model.

\section*{A more general derivation}

The derivation conducted in the last section relies on a linearization of the fitness function, which allows us to express all quantities of interest as a function of the mean recombination activity $R$, and thus express the self-consistent mean-field argument directly on $R$. However, this linear approximation is not well suited to all fitness models. In this section, we propose an alternative, more general, derivation of the self-consistent mean-field argument, which can be used for arbitrary fitness schemes. This derivation can also be generalized to more complex models, in particular, allowing for variation in recombination rates across hot spots (next section). The whole derivation is done only under the polymorphic regime.

Consider a typical PRDM9 allele, with frequency $x(t)$ and activity $\theta(t)$ at time $t$. As we have seen above, the evolution of the allele frequency and activity are coupled with each other. However, we can reparameterize the problem, so as to eliminate $x(t)$, as follows.

First, as seen in the last section, the activity of the allele decays at a rate proportional to the allele's frequency:
\begin{eqnarray*}
\frac{d\theta}{dt} &=& - \rho x \theta.
\end{eqnarray*}
This equation has general solution:
\begin{eqnarray}
\label{intrinsictheta}
\theta(t) &=& e^{-\rho \int_0^t x(u) du}
= e^{-\rho \xi(t)}
\end{eqnarray}
where we define
\begin{eqnarray*}
\xi(t) &=& \int_0^t x(u) du.
\end{eqnarray*}
The variable $\xi$ can be seen as the \emph{intrinsic age} of the allele. It accounts for the fact that an allele ages more rapidly if it segregates at a higher frequency. 
What equation \ref{intrinsictheta} tells us is that the activity of the allele at a given time point only depends on the allele's intrinsic age. We can therefore parameterize all quantitites associated with a PRDM9 allele directly as a function of its intrinsic age, thus dispensing with the need to solve $x(t)$ as a function of time. In fact, instead of the intrinsic age $\xi$, it is more convenient to reparameterize the problem in terms of the dimensionless variable:
\begin{eqnarray*}
z(t) = \rho \xi(t) &=& \rho \int_0^t x(u) du.
\end{eqnarray*}
As we have seen in the last section, the mean time between successive invasions is approximately equal to:
\begin{eqnarray*}
\tau &=& \int_0^\infty x(u) du = \xi(\infty),
\end{eqnarray*}
which reveals that $\tau$ can also be interpreted as the mean intrinsic longevity of PRDM9 alleles. Equivalently, $z(\infty) = \rho \tau$, and thus, $z(t)$ is a monotonous function of time, varying from 0 to $\rho \tau$. The inverse of this function is noted $t^*(z)$.
In the following, all quantities expressed in the $z$-parameterization will be indicated by a star. In particular, we define:
\begin{eqnarray*}
x^*(t) &=& x(t^*(z))
\end{eqnarray*}
and 
\begin{eqnarray*}
\theta^*(z) &=& \theta(z(t)) = e^{-z}
\end{eqnarray*}
In the $z$-parameterization, the activity of a new-born allele is thus $\theta^*(0) = 1$, and the activity of a dying allele is equal to $\theta_\infty = \theta^* (\rho \tau) = e^{-\rho \tau}$.
Other quantities will be reparameterized below as a function of $z$.

Some elementary relations between the $t$- and $z$-parameterizations can be pointed out, which will be useful in the following. First, 
\begin{eqnarray*}
\frac{dz}{dt} &=& \rho x
\end{eqnarray*}
and second
\begin{eqnarray*}
\frac{dx^*}{dz} &=& \frac{dx}{dt} / \frac{dz}{dt} = sx / \rho x = \frac{s^*(z)}{\rho}
\end{eqnarray*}
where we have defined $s^*(z) = s(\theta(t))$, the selection coefficient associated with the allele, in the $z$-parameterization.

The fundamental meaning of the tiling argument is this: suppose we randomly sample an allele at stationarity. Then, the probability that this allele has intrinsic age $\xi$ is uniformly distributed between $0$ and $\tau$ (or, equivalently, $z(t)$ is uniformly distributed between $0$ and $\rho \tau$). As a consequence, we can approximate any average over the population, at stationarity, as an integral over $z$. That is, for any function $q(x,\theta)$:
\begin{eqnarray*}
\sum_i x_i q(x_i,\theta_i) &\simeq& \frac{1}{\tau} \int x(t) q(x(t),\theta(t)) dt
\\
&=& \frac{1}{\rho \tau} \int_0^{\rho \tau} q(x^*(z),\theta^*(z)) dz
\end{eqnarray*}
Using this trick, we can express several key summary statistics, all of which are averages over the population at stationarity.
Thus, the mean recombination activity is given by:
\begin{eqnarray*}
R = \sum_i x_i \theta_i &=& \frac{1}{\rho \tau} \int_0^{\rho \tau} \theta^*(z) dz
= \frac{1}{\rho \tau} \int_0^{\rho \tau} e^{-z} dz
= \frac{1 - e^{-\rho \tau}}{\rho \tau}
\end{eqnarray*}
The diversity:
\begin{eqnarray*}
D^{-1} = \sum_i x_i^2 = \frac{1}{\rho \tau} \int_0^{\rho \tau} x^*(z) dz 
= - \frac{1}{\rho^2 \tau} \int_0^{\rho \tau} s^*(z) z dz
\end{eqnarray*}
where we have integrated by parts, using the relation $dx^* / dz = s^*(z)/\rho$.
Since the turnover time is equal to $T = \tau D$, this can also be rewritten as:
\begin{eqnarray*}
T = (\tau \D)^{-1} &=& - \frac{1}{(\rho \tau)^2} \int_0^{\rho \tau} s^*(z) z d z 
\end{eqnarray*}
The fitness of an allele of activity $\theta_0 = \theta^*(z_0)$ can be calculated by averaging over all possible diploid backgrounds:
\begin{eqnarray*}
w^*(z_0) = w(\theta_0) = \sum_i x_i f \left(  \frac{\theta_0 + \theta_i}{2} \right)
&=& \frac{1}{\rho \tau} \int_0^{\rho \tau} f \left(  \frac{\theta^*(z_0) + \theta^*(z)}{2} \right) dz
\end{eqnarray*}
and the mean fitness over the population is obtained by a double averaging procedure:
\begin{eqnarray*}
\bar{w} &=& \frac{1}{\rho \tau} \int_0^{\rho \tau} w^*(z) dz
= \frac{1}{(\rho \tau)^2} \int_0^{\rho \tau} \int_0^{\rho \tau} f \left(  \frac{ \theta^*(z_1) +  \theta^*(z_2)}{2} \right) d z_1 d z_2
\end{eqnarray*}
From these equations, the selection coefficient acting on an allele of age $z$, noted $s^*(z)$, is given by:
\begin{eqnarray*}
s^*(z) &=& \frac{w^*(z) - \bar w}{\bar w}
\end{eqnarray*}
and in particular, the selection coefficient for a newborn allele is:
\begin{eqnarray*}
s_0 = s^*(0) &=& \frac{w^*(0) - \bar w}{\bar w}
\end{eqnarray*}
The quantities $R$, $T$, $w^*$ and $s^*$ only depend on the fitness function $f$ and the compound parameter $\rho \tau$. This parameter, $\rho \tau$, is now our key parameter to be determined through a self-consistent argument. As was done in the previous section, we can obtain an independent derivation for $\tau$, as the inverse of the rate of invasion of the population by new alleles:
\begin{eqnarray*}
\tau^{-1} = \lambda &=& 4 N_e u s_0,
\end{eqnarray*}
or, equivalently,
\begin{eqnarray*}
\rho \tau &=& \frac{4 N_e v r_0}{4 N_e u s_0} = \frac{v r_0}{u s_0}
\end{eqnarray*}
Using this relation, the self-consistent mean-field argument can now be most conveniently expressed on the selection coefficient $s_0$: this selection coefficient is an integral relation that depends on $\rho \tau$, which in turn depends on $s_0$. The self-consistent solution to this system of two equations can obtained numerically (see methods).

\section*{Allowing for variation among hot spots in recombination rate}

Thus far, the model assumes that all active hot spots (i.e. not yet subjected to biased gene conversion) have the same recombination rate $r_0$. In reality, there is a substantial variance across hot spots in their recombination rate. In this section, we explore a generalization of the red queen model that relaxes this uniformity assumption.

Specifically, we assume that the distribution of recombination rates across the hot spots associated with a new born PRDM9 allele is a gamma distribution, of mean $r_0$ and shape parameter $a$ (with smaller value of $a$ inducing  higher variance across hot spots). Equivalently, the recombination rate of a given hot spot is equal to $r_0 c$, where $c$ is the relative recombination rate. The distribution of $c$ is thus a gamma of mean 1 and shape parameter $a$:
\begin{eqnarray*}
\phi(c) &=& \frac{b^a}{\Gamma(a)} c^{a-1} e^{-ac}.
\end{eqnarray*}
%\begin{eqnarray*}
%\phi(r) &=& \frac{b^a}{\Gamma(a)} r^{a-1} e^{-br}.
%\end{eqnarray*}
Consider a typical PRDM9 allele. For this allele, we define $\theta_c(t)$ as the fraction of hotspots with recombination rate $r_0 c$ that are active at time $t$. The rate of erosion for this specific subset of hotspots is given by:
\begin{eqnarray*}
\frac{d \theta_c}{d t} &=& - \rho c x \theta.
\end{eqnarray*}
By a similar argument as the one used in the last section, this has solution:
\begin{eqnarray*}
\theta_c(t) &=& e^{-\rho c \xi(t)} = e^{-c z(t)}
\end{eqnarray*}
or, more directly in the $z$-parameterization:
\begin{eqnarray*}
\theta^*_c(z) &=& e^{-c z}
\end{eqnarray*}
The total (relative) recombination activity induced by a PRDM9 allele with intrinsic age $z$ is then obtained by averaging $\theta^*_c$ over $c$ (under the gamma), which is analytical:
\begin{eqnarray}
\label{powertheta}
\theta^*(z) &=& \int c \theta^*_c(z) \phi(c) dc = \left( \frac{a}{a + z} \right)^{a+1}
\end{eqnarray}
From there, the overall derivation is essentially the same as in the previous section, with the only difference that the integral equations for $w^*$ and $s^*(0)$ are now computed using the expression for $\theta^*(z)$ given by equation \ref{powertheta}, instead of equation \ref{intrinsictheta}. These expressions are, as before, functions of $\rho \tau = \frac{u r_0}{v s^*(0)}$, again giving a self-consistent relation on $s^*(0)$ which can be solved numerically.

As before, once the self-consistent solution is found, the summary statistics at the equilibrium set-point can be computed by averaging over $z$. In particular, the mean recombination activity over the population is now:
\begin{eqnarray*}
R = \frac{1}{\rho \tau} \int_0^{\rho \tau} \theta^*(z) d z &=& \frac{1}{\rho \tau} \left[ 1 - \left( \frac{a}{a + \rho \tau} \right)^a \right]
\end{eqnarray*}
In addition to the mean recombination rate $R$, the mean fraction of active allele-specific targets that are still active, which we call $H$, is another statistic of interest (for which direct empirical evidence is available, based on Chip-Seq experiments on hybrids). Thus far, without variance across hot spots in their basal recombination rate, the two statistics $R$ and $H$ were identical. However, this is not anymore the case under the gamma model now considered. The fraction of active targets for an allele of age $z$, $h^*(z)$ is the mean of $\theta_c(z)$ over the gamma distribution:
\begin{eqnarray*}
h^*(z) &=& \int \theta^*_c(z) \phi(c) dc = \left( \frac{a}{a + z} \right)^{a}
\end{eqnarray*}
and the average over the population is, again, obtained by averaging over $z$ :
\begin{eqnarray}
H = \frac{1}{\rho \tau} \int_0^{\rho \tau} h^*(z) d z
\end{eqnarray}
This is equal to:
\begin{eqnarray}
H &=& \frac{1}{\rho \tau} \left[ 1 - \frac{a}{a-1}  \left( \frac{a}{a + \rho \tau} \right)^{a-1} \right]
\end{eqnarray}
for $a \neq 1$, and to:
\begin{eqnarray}
H &=& \frac{\ln 1 + \rho \tau}{\rho \tau}
\end{eqnarray}
for $a = 1$ .
Finally, the expressions for $D$ and $T$, as integrals depending on $s^*(z)$, are unchanged.



%The development just conducted assumes that the fitness depends on the mean recombination rate. Alternatively, it might be reasonable to assume that the fitness depends on the relative number of recombination hot spots that are still active. The fraction of active hotspots induced by a given allele of intrinsic age $z$ is given by integrating $\theta_c$, instead of $c \theta_c$, over the gamma distribution:
%\begin{eqnarray*}
%\label{powerthetafraction}
%\theta^*(z) &=& \int \theta^*_c(z) \phi(c) dc = \left( \frac{a}{a + z} \right)^{a}
%\end{eqnarray*}
%and the average number of active hot spots over the population is, again, obtained by averaging over $z$ which, for %$a \neq 1$, gives:
%\begin{eqnarray}
%R = \frac{1}{\rho \tau} \int_0^{\rho \tau} \theta^*(z) d z &=& \frac{1}{\rho \tau} \left[ 1 - \frac{a}{a-1}  \left( \frac{a}{a + \rho \tau} \right)^{a-1} \right]
%\end{eqnarray}
%and for $a = 1$ is equal to:
%\begin{eqnarray}
%R = &=& \frac{\ln \rho \tau}{\rho \tau}
%\end{eqnarray}
%From there, calculating the self-consistent solution under this variant of the model can be done by simply using equation \ref{powerthetafraction} instead of equation \ref{powertheta} in the numerical integrals leading to the estimate of $s_0$.

\section*{General scaling behavior}

\subsection*{Scaling relations as a function of $N_e$}

Even without any explicit or any numerical solution for $s$, the relations derived above for $R$, $\tau$, $s$, $T$ and $D$ imply the following scaling relations for all of these quantities as a function of $N_e$. First, $s$ is the solution of a self-consistent equation that depends only on $\frac{v r_0}{u}$, and therefore does not depend on $N_e$. The same thing is true more generally for $w^*(z)$, $\bar w$ and $s^*(z)$ under arbirtary values of $z$, for $\rho \tau$, and thus also for all quantities that depend only on $\rho \tau$: in particular $R$ and $T$. Since $\rho = 4 N_e v r_0$ itself is directly proportional to $N_e$ and $\rho \tau$ does not depend on $N_e$, $\tau$ is inversely proportional to $N_e$. In turn, since $T = \tau D$ is constant as a function of $N_e$, $D$ is proportional to $N_e$. 

All of those relations are valid in the polymorphic regime. In the succession regime, the main difference is that $D=1$, by definition. Otherwise, $R$, $\tau$ and $s$ are still constant as a function of $N_e$, and $T$ is now inversely proportional to $N_e$.


\subsection*{First-order development ($\rho \tau << 1$)}

Throughout the self-consistent derivation presented in the last sections, the compound parameter $\rho \tau$ plays a key role. This parameter can be re-expressed as:
\begin{eqnarray*}
\rho \tau = \frac{v r_0}{u s} = \frac{4 N_e v r_0}{4 N_e u s_0}
\end{eqnarray*}
which is the ratio of the two antagonistic forces of the Red Queen process, the rate of erosion of PRDM9 targets by dBGC ($4 N_e v r_0$) and the rate of invasion by new PRDM9 variants ($4 N_e u s_0$). The equilibrium set-point of the Red Queen process, in terms of the realized value of $\R$, is essentially detemined by the relative magnitude of these two rates. Thus, if $\rho \tau >> 1$, erosion is strong compared to elicitation of new PRDM9 alleles, and $\R$ is expected to be small. Conversely, if $\rho \tau << 1$, $\R$ is close to 1. All solutions presented above of $R$ as a function of $\rho \tau$ are indeed decreasing functions, ranging from 0 to 1 as $\rho \tau$ varies from 0 to $+\infty$.

The case where $\rho \tau << 1$ lends itself to a simple first-order approximation of the self-consistent mean-field solution. This approximation may not be strictly valid in the empirically relevant regimes, where $\rho \tau$ seems instead to be of the order of 1 (see below). Nevertheless, it leads to simple expressions for all summary statistics of interest in the stationary regime, thus giving a very concise (if somewhat idealized) summary of the scaling of the stationary regime of the Red Queen as a function of the parameters of the model.

To derive this approximation, we start by recalling the integral expression for the fitness of an allele of activity $\theta^*(z_0)$:
\begin{eqnarray*}
w^*(z_0) &=& \frac{1}{\rho \tau} \int_0^{\rho \tau} f \left(  \frac{\theta^*(z_0) + \theta^*(z)}{2} \right) dz
\end{eqnarray*}
Then, defining
\begin{eqnarray*}
\alpha &=& |f'(0) (\theta^*)'(0)|
\end{eqnarray*}
making a first-order Taylor development of $f$ at 0:
\begin{eqnarray*}
f \left(  \frac{\theta^*(z_0) + \theta^*(z)}{2} \right)  &\simeq& f(0) - \frac{\alpha}{2} (z_0 + z) = 1 - \frac{\alpha}{2} (z_0 + z) 
\end{eqnarray*}
and integrating with respect to $z$ gives an approximation for $w^*(z_0)$:
\begin{eqnarray*}
w^*(z_0) &\simeq& \frac{1}{\rho \tau} \int_0^{\rho \tau} 1 - \frac{\alpha}{2} (z_0 + z) dz
= 1 - \frac{\alpha z_0}{2} - \frac{\alpha \rho \tau}{4}
\end{eqnarray*}
Similarly, the mean fitness over the population is approximately equal to:
\begin{eqnarray*}
\bar{w} &=& \frac{1}{\rho \tau} \int_0^{\rho \tau} w^*(z) dz
= \frac{1}{(\rho \tau)^2} \int_0^{\rho \tau} \int_0^{\rho \tau} f \left(  \frac{ \theta^*(z_1) +  \theta^*(z_2)}{2} \right) d z_1 d z_2
\\
&\simeq&
 \frac{1}{(\rho \tau)^2} \int_0^{\rho \tau} \int_0^{\rho \tau} f \left(  1 - \frac{\alpha}{2} (z_1 + z_2) \right) d z_1 d z_2
\\
&=& 1 - \frac{\alpha \rho \tau}{2}
\end{eqnarray*}
The selection coefficient can then be calculated:
\begin{eqnarray*}
s^*(z_0) = \frac{w^*(z_0) - \bar w}{\bar w} &\simeq& \frac{\alpha \rho \tau}{4}  - \frac{\alpha z_0}{2}
\end{eqnarray*}
In particular, for $z_0 = 0$:
\begin{eqnarray*}
s^*(0) = \frac{w^*(0) - \bar w}{\bar w} &=& \frac{\alpha \rho \tau}{4} = \frac{\alpha v r_0}{4 u s^*(0)} 
\end{eqnarray*}
or, equivalently
\begin{eqnarray*}
s^*(0)^2 &=& \frac{\alpha \rho \tau}{4} = \frac{\alpha v r_0}{4 u} 
\end{eqnarray*}
which yields an explicit solution for $s^*(0)$:
\begin{eqnarray*}
s^*(0) &=& \frac{1}{2} \sqrt \frac{\alpha v r_0}{u} 
\end{eqnarray*}
This can then be used to get an explicit expression for $\rho \tau$:
\begin{eqnarray*}
\rho \tau = \frac{v r_0}{u s} &=& \frac{v r_0}{u s} \sqrt \frac{4 u}{\alpha v r_0}  = 2 \sqrt \frac{v r_0}{u \alpha} 
\end{eqnarray*}
and for $R$:
\begin{eqnarray*}
R = \frac{1 - e^{-\rho \tau}}{\rho \tau} &\simeq& 1 - \frac{1}{2} \rho \tau = 1 - \sqrt \frac {v r_0 }{u \alpha}
\end{eqnarray*}

Concerning $D$, the integral expression is:
\begin{eqnarray*}
D^{-1} &=& - \frac{1}{\tau} \frac{1}{\rho \tau} \int_0^{\rho \tau} s^*(z) z dz
\end{eqnarray*}
which, using the approximation given above for $s^*(0)$ and after some algebra gives:
\begin{eqnarray*}
D^{-1} = \frac{s^*(0)}{6 \tau}
\end{eqnarray*}
Noting that $\tau = 4 N u s^*(0)$, this yields a surprisingly simple scaling identity for $D$:
\begin{eqnarray*}
D &\simeq& 24 N_e u
\end{eqnarray*}
Finally, 
\begin{eqnarray*}
T = D \tau = D \rho \tau / \rho \simeq \frac{24 N_e u}{4 N_e v r_0} 2 \sqrt \frac{v r_0}{u \alpha}  = 12 \sqrt \frac{u}{v r_0 \alpha}
\end{eqnarray*}

In the end, defining the compound parameter $\epsilon = \frac{v r_0}{u}$, all of the scaling relations just derived can be expressed in terms of 2 scaling factors, $\epsilon$ and $N_e u$. The relations are then as follows (table 1): in the polymorphic regime, $\rho \tau$ and $1 - R$ scale as $\sqrt \epsilon$, $s$ and $T$ as $\frac{1}{\sqrt \epsilon}$, and $D$ as $N_e u$. In the succession regime, the only diffrence is that $D=1$ and that $T$ now scales as  $\frac{1}{N_e u \sqrt{\epsilon}}$.

\section*{Comparing analytical approximations with simulation results}

The scaling relations predicted for the four summary statistics by our mean-field approximation are shown on the top of the simulation results on figures 2, 3, 4 and 5, under the small-load approximation (green) as well as for arbitrary $\epsilon$, by numerically solving the self-consistent equation \ref{selfconsistent} (orange).
In spite of the rather bold approximations that were made, the predicted scaling is globally in accordance with the simulation results. Expectedly, the small-load approximation fails when the recombination gap becomes significant (typically, when it is larger than 0.2).

\section*{Discussion}

In this work, we have presented a combination of simulations and analytical approximations, giving new insights into the population genetics of the Red Queen model of recombination. Our more specific focus was on achieving a better understanding of how the interplay between mutational input, positive selection and dBGC modulates the statistical patterns of PRDM9 diversity and recombination landscapes. In the end, our analysis yields a global picture of the of qualitative regimes and the scaling behavior of the Red Queen model, as a function of effective population size and the parameters of the genetic system. 

\subsection*{Evolutionary regimes and scaling behavior}

The stationary regime of the Red Queen process can be characterized in terms of the mean recombination activity $R$ (or, equivalently, the mean level of erosion $H$), the characteristic time of turnover of recombination landscapes $T$,  the strength of selection acting on new PRDM9 alleles, and the genetic diversity at the PRDM9 locus $D$. The overall scaling behavior of all of these quantities as a function of the parameters of the model is captured in a particularly compact form in the weak erosion regime, such as summarized in table 1. Those relations are less accurate in more intense regimes, although they correctly describe the qualitative behavior of the process even in those cases.

First, the mean level of erosion $H$ and the strength of selection at stationarity $s$ do not depend on effective population size $N_e$. Both $H$ and $s$ increase with the mutation rate $v$ and the conversion strength $g$ at the targets and decrease with the mutation rate $u$ and the strength of selection acting on PRDM9. Similar observations concerning the role of mutation rates at the targets and at the PRDM9 locus were previously reported based on simulation analyses (Ubeda ..). Here, we give a more quantitative estimate of this scaling, as the square root the compound parameter $\epsilon = \frac{vg}{u}$.

Second, in terms of the equilibrium genetic diversity $D$ at the PRDM9 locus, the Red Queen has two distinct regimes. For small $N_e u$, the process is characterized by a succession of hard sweeps of PRDM9 alleles invading the entire population ($D=1$ at all times, except during the sweeps). In contrast, for large $N_e u$, polymorphism is maintained at the PRDM9 locus. The transition between the two regimes can be visualized by imagining that we progressively increase the effective population size under a fixed mutation rate $u$. In the succession regime (small $N_e$), the Red Queen merely accelerates (i.e. the mean time between successive sweeps decreases) with $N_e$, up until the point where successive sweeps start to overlap. At this point, the Red Queen process enters the polymorphic regime. Further increasing $N_e$, the turnover time now stays constant, and the Red Queen, instead of accelerating, spreads over an increasing number of segregating PRDM9 alleles and, correlatively, recombination is distributed over an increasing number of weaker hot spots. 

An intuitive interpretation of those scaling identities can be given be as follows. First, the equilbrium recombination activity is determined by the relative magnitude of the two forces of the Red Queen, which is captured by $\rho \tau = \frac{4 N_e v g}{4 N_e u s} = \epsilon/s$. The effective population size cancels out from this ratio, and thus the equilibrium set point does not depend on $N_e$.  

Second, the scaling is not linear, but only as the square root of $\epsilon$, because of the feedback between the two arms of the Red Queen. For instance, an increase in $\epsilon$ (e.g. through an increase in the mutation rate at the targets $v$) will result in a stronger erosion by dBGC. However, this increased depletion of recombination will in turn trigger an increase in the strength of positive selection on new PRDM9 alleles (captured by $s$), thus partially restoring higher levels of recombination. As a result of this feedback, the response of $\rho \tau = \frac{\epsilon}{s}$, and thus of $R$, to this change in the value of $\epsilon$ is less than linear (being in fact equally distributed between the two arms in the small-gap limit, hence the square root scaling).

Third, in the polymorphic regime, mean recombination activity and PRDM9 diversity are essentially uncoupled -- the first depends only on $\epsilon$ and the second only on $N_e u$. This is because the mean recombination activity implied by a given value of $\epsilon$ can be equivalently realized, either by few PRDM9 alleles segregating at high frequencies, eroding their targets at a high rate and thus quickly replaced by other alleles, or by many alleles segregating at low frequencies, eroding their targets at a lower rate and replaced less often. These two extreme regimes can essentially be seen as two equivalent 'assembly lines', which merely differ in their latency but not in their output (a figure here). The choice between these alternative implementations of the equilbrium is then essentially determined by the mutational input at the PRDM9 locus, $N_e u$ -- thus, in a sense, neutrally.

This last point is of particular importance concerning the correct interpretation of the empirically observed patterns of polymorphism and divergence at the PRDM9 locus: positive selection on PRDM9 may easily reach very high levels under reasonable parameter regimes of the Red Queen process, which will easily translate into high ratios of non-synonymous to synonymous rates. On the other hand, this type of positive selection, even when very strong, does not promote genetic diversity within populations -- it is not diversifying. At least under the assumptions of the models explored here, mutation, not selection, explains the intra-specific genetic diversity of PRDM9.

\subsection*{Empirical calibration}

The model can be tentatively calibrated against current empirical evidence. This can most easily be done using the mouse as our model species, while keeping an eye on data from humans as an additional check at lower $N_e$. 

The three mouse subspecies, castaneus, domesticus and musculus, have diverged from each other around .. . The effective population size is estimated to be of the order of $N_e = 10^5$ (ranging from $5.10^4$ in musculus up to $5.10^5$ in castaneus, Phifer-rixey et al, 2012). Each subspecies has its own major PRDM9 allele, segregating at a frequency of the order of 30\%, with most other alleles being much rarer (Buard et al, 2010). The levels of heterozygosity is around 0.2 to 0.5, thus suggesting a value of $D$ between 5 and 10 in each subpopulation. 

Experiments on hybrids between subspecies show that the major allele of each subspecies has eroded a fraction of its targets of the order of 50\% (40\% in castaneus, Baker et al, see other data from Davies et al). Finally, patterns of non-synonymous versus synonymous variation clearly suggest the presence of strong positive selection acting on the zinc-finger array of PRDM9. The complicated process of concerted evolution acting on this minisatellite makes it difficult to reliably estimate the strength of selection using classical codon models. Nevertheless, the $dN/dS$ is of the order of 10, with most non-synonymous changes specifically concentrated on the 3 key codons determining the DNA binding specificity of zinc-fingers 8-12. These observations suggest a scaled selection coefficient well above 1 $S = 4 N_e s >> 1$, which can be reasonably assumed to be of the order of 10 or above.

By comparison, in humans, effective population size is classically estimated at $10^4$, although it might have been slighlty larger (of the order of $3.10^4$) in deeper evolutionary times. The PRDM9 locus is slightly less diverse than in mouse, with the major PRDM9 allele segregating at a frequency of around 50\% in africans, suggesting a value of $D$ of the order of 3. Positive selection is also detected on PRDM9, again with a dN/dS of the order of 10. 

Assuming a point mutation rate of $10^{-8}$ and an effective number of 10 inactivating mutations per target, $v$ can be estimated to be of the order of $10^{-7}$. DBS initiation maps inferred by Chip Seq suggest that the distribution of DSB initiation is approximately exponential ($a=1$). This leaves us with 3 parameters to estimate ($u$, $g$ and $\alpha$), based on three summary statistics ($D \simeq 7$, $4Ns = 10$ to 40, and $H \simeq 0.5$).

Our simulation experiments suggest that these summary statistics can be well reproduced for values of $u$ between $3.10^{-6}$ and $10^{-5}$, $g = 3.10^{-3}$ to $10^{-2}$. 

The calibration of the model turns out to be rather constrained. On the one hand, the presence of polymorphism ($D>1$), which scales as $N_e u$, implies that $u$ should not be too small. Similarly, the presence of positive selection at the PRDM9 locus ($S = 4 N_e s >> 1$) implies that $s$ should not be too small either. Thus, the product $us$ is bounded from below. On the other hand, and irrespective of the exact fitness model, the mean level of erosion at equilibrium is a function of the compound parameter $\rho \tau = \frac{v g}{u s}$. In practice, the significant levels of depletion observed in several mouse subspecies, of the order of 50\%, imply that this compound quantity should be of the order of 1 -- or equivalently, $v g \simeq u s$. Since the mutation rate at the targets $v$ is small, this then means that the intensity of dBGC, $g$, should be sufficiently high.

Quantitatively, with $D \simeq 7$ and $N_e = 10^{5}$, $u$ should be of the order of $10^{-5}$. Similarly, a scaled selection coefficient $4 Ns$ of the order of 10 or more implies that $s$ should not be much smaller than $10^{-4}$. Thus, $us \simeq vg $ should be of the order of $10^{-9}$. Assuming a point mutation rate of $10^{-8}$ and an effective number of 10 inactivating mutations per targets, $v$ is of the order of $10^{-7}$. With these constraints, in order to explain the observed levels of erosion, $g$ should not be much smaller than $10^{-2}$. 

The constraints just mentioned are relatively general. They do not depend on the exact fitness model. In fact, once those constraints are fulfilled by an empirically reasonable combination of parameter values, it still remains to fine tune the parameters of the fitness model, so as to find a regime that effectively realizes the correct equilibrium set point, in terms of the implied values of $R$ and $s$. Here, using a value of $a = 1$ for the variance across hot-spots in DSB initiation rate, we can at least qualitatively estimate the $\alpha$ parameter of the fitness model (of the order of 10). Under this fitness function, the levels of depletion that are necessary to cause a drop in fertility of 10\% are of the order of ...

Depending on the exact fitness model, allowing for a lower $g$ either would lead to equilbrium levels of erosion that are much too low compared to empirically observed levels, or would push the Red Queen into the weak selection regime in the mouse, and, by extrapolation, to an effectively neutral regime in species with smaller effective population sizes such as humans. The idea that the hot spot parardox could be resolved by a neutral model, with sufficiently high mutation rate for PRDM9, has already been suggested previously (Jeffreys). However, this is in contradiction with the empirical evidence for relatively strong positive selection acting on PRDM9.

\subsection*{Perspectives}

The strong empirical constraints mentioned above, in particular concerning the high value for $u$, raise the question of why the key trans-acting factor of the Red Queen of recombination happens to have such a high mutation rate in the first place. An interesting possibility would be that PRDM9 has been coopted in this intra-genomic Red Queen precisely for that reason. Altermatively, it is possible to imagine that the detailed genetic structure of the zinc-finger domain has been progressively optimized by a higher-level evolutionary process. For instance, the number of zinc-finger is both correlated with the mutation rate 

A spontaneous evolution of PRDM9 toward a higher mutation rate could for instance be imagined as follows: the 

These interesting speculations could be further investigated in the context of the present modeling framework.

Another intriguing aspect of PRDM9 is its role in hybrid sterility. PRDM9 was in fact initially discovered precisely in this context. Recent developments have suggested that the hybrid sterility phenotype is directly related to the macroscopic asymmetry in the patterns of PRDM9 binding along chromosomes of meiotic cells in F1 hybrids. In turn, this asymmetry is a direct consequence of the differential erosion of PRDM9 targets along the two parental chromosomes, itself due to distinct major PRDM9 alleles segregating in the two subpopulations of origin (refs). Altogether, these observations point toward a potentially important role for the Red Queen process of recombination turnover in the creation of genetic barriers between subspecies. The modeling framework introduced here could certainly be extended in a metapopulation context, so as to further investigate those particularly interesting questions.

Many of the developments presented here appear to be fairly general. In particular, the scaling as a function of $N_e$ is essentially valid under arbitrary fitness models, as long as the absolute fitness of a diploid can be expressed as a function of the distribution activities (fraction of active targets) of its two alleles. Note that, although we have used an additive model for the relation between recombination and PRDM9 diploid genotype, the mean-field developments and the scaling properties presented here do not require this additivity assumption.


On the other hand, more complex situations could be imagined. For instance, 

our model predicts that mean recombination activity stays constant as $N_e$ increases, but, owing to a larger PRDM9 diversity, recombination spreads out over a larger number of hotspots across the genome. At the level of the population, this lesser variance of recombination ladnscapes results in a fragmentation of haplotype blocks into smaller independent haplotypes, and thus. The prediction is thus that larger effective population sizes should result in less intense background selection. How this effect could play on the fitness of PRDM9 alleles should be further investigated. Intuitively, rare PRDM9 alleles should be favored precisely because they break up haplotype blocks that tend to be under strong LD at the level of the population. The fitness of a PRDM9 allele should depend, not just on the mean recombination activity in the population, but also on the variance. This would change the predictions of the model.


Other possible developments: Here, assumption of a panmictic population. However, PRDM9 plays in a complex manner in hybrids. Thus, interesting to extend those simulation experiments and theoretical developments in the context of a meta-population model (complex of species), so as to more thoroughly investigate the potential role of PRDM9 gene as a speciation gene.

Theoretical aspects: explore the weak selection regime. Those theoretical developments could be used in many other contexts, in particular for ecological red queen situations other than intra-genomic conflicts.


Coupling these arguments with gBGC: if $b_0$ and $v/u$ are constant across mammals, then the number of GC substitutions per gBGC episode (per hot spot) approx. constant across mammals. On the other hand, the number of gBGC episodes per My and per Mb scales linearly with $N_e u$. This predicts the following affine relation between equilibrium GC bias and $N_e u$;
\begin{eqnarray*}
\frac{GC^*}{AT^*} = a + b N_e u
\end{eqnarray*}
Note that this affine scaling is different from, and might be more reasonable than, the exponential scaling expected under constant $b_0$ and uniform and constant recombination landscapes across mammals.

Another consequence: we have an estimate of the scaled selection coefficient $4 N_e s$ at the PRDM9 locus. We could use it to derive an estimate of the rate of adatpive substitution at the PRDM9 locus, relative to the neutral rate.
In the context of classic codon models, $dN/dS$ can be seen as an empirical estimate of the relative fixation probability of non-synonymous substitutions: $\omega = \langle 2 N_e p_{fix} \rangle$. In a context of strong positive selection, $p_{fix} = 2s$ and thus $\omega = 4 N_e s$. By analogy, the relative rate of adaptive evolution or PRDM9 would be:
 \begin{eqnarray*}
\frac{\lambda}{\lambda_0} &=& \frac{4 N_e s}{\D} = \frac{4 N_e s}{24 N_e u} = \frac{s}{6 u} = \frac{\alpha \epsilon}{12 u}.
\end{eqnarray*}
Here, the rate of adaptive substitution should be divided by $D$, since we should account for the fact that each new adaptive substitution at the PRDM9 locus is due to an invasion by a new allele, which is a mutant of one out of an effective number of $D$ alleles segregating at this locus at any given time.


\newpage

%\bibliographystyle{plain}
%\bibliography{red-queen}

\part*{Figures}

\begin{figure*}[!ht]
	  \centering
       \includegraphics[width=0.8\textwidth]{Images/simulated-trajectory.pdf}\\
\end{figure*}
\textbf{Figure 1.} A typical simulation trajectory of the Red Queen model, with allele frequencies (top) and allele activity (bottom) through time.

\newpage

\begin{figure*}[!ht]
	  \centering
       \includegraphics[width=0.8\textwidth]{Images/mean-activity.pdf}\\
\end{figure*}
\textbf{Figure 2.} Mean activity $\R$, as a function of effective population size $N_e$ (top left), mutation rate at the targets $v$ (top right), mutation rate at the PRDM9 locus $u$ (bottom left) and selection strength $\alpha$ (bottom right). Mean-field approximation (orange) and small-load approximation (green).

\newpage

\begin{figure*}[!ht]
	  \centering
       \includegraphics[width=1.0\textwidth]{Images/prdm9-diversity.pdf}\\
\end{figure*}
\textbf{Figure 3.} Diversity at the PRDM0 locus $\D$, as a function of effective population size $N_e$ (top left), mutation rate at the targets $v$ (top right), mutation rate at the PRDM9 locus $u$ (bottom left) and selection strength $\alpha$ (bottom right). Mean-field approximation (orange) and small-load approximation (green).

\newpage

\begin{figure*}[!ht]
	  \centering
       \includegraphics[width=1.0\textwidth]{Images/landscape-variance.pdf}\\
\end{figure*}
\textbf{Figure 4.} Relative variance of recombination landscapes $\V$, as a function of effective population size $N_e$ (top left), mutation rate at the targets $v$ (top right), mutation rate at the PRDM9 locus $u$ (bottom left) and selection strength $\alpha$ (bottom right). Mean-field approximation (orange) and small-load approximation (green).

\newpage

\begin{figure*}[!ht]
	  \centering
       \includegraphics[width=1.0\textwidth]{Images/turn-over.pdf}\\
\end{figure*}
\textbf{Figure 5.} Turnover time $\T$, as a function of effective population size $N_e$ (top left), mutation rate at the targets $v$ (top right), mutation rate at the PRDM9 locus $u$ (bottom left) and selection strength $\alpha$ (bottom right). Mean-field approximation (orange) and small-load approximation (green).

\newpage

\begin{figure*}[!ht]
	  \centering
       \includegraphics[width=0.8\textwidth]{Images/fixedpoint.pdf}\\
\end{figure*}
\textbf{Figure 6.} Self-consistent solution as a fixed point, in the form $R = g(R)$, for two values of $\epsilon = \sqrt{\dfrac{v r_0}{\alpha u}}$.

%\newpage

%\textbf{Figure 6.} Allele frequency ($x_t$) as a function of activity ($\theta_t$).

\newpage

\begin{figure*}[!ht]
	  \centering
       \includegraphics[width=0.8\textwidth]{Images/single-allele.pdf}\\
\end{figure*}
\textbf{Figure 7.} Trajectory of the frequency and the activity of a typical allele through time, under the deterministic and mean-field approximation (see text for details).

%\newpage

%\textbf{Figure 6.} Allele frequency ($x_t$) as a function of activity ($\theta_t$).

\newpage

\begin{figure*}[!ht]
	  \centering
       \includegraphics[width=0.8\textwidth]{Images/tilling-argument.pdf}\\
\end{figure*}
\textbf{Figure 8.} A tiling principle. On the left, an idealized realization of the Red Queen dynamic is depicted, in which new alleles invade the population at regular time intervals, all equal to $\tau$, and have the same allele frequency trajectory (such as depicted in figure 6). Taking the sum of allele frequencies at any given time (vertical bar), which by definition is equal to 1, is then equivalent to summing the values taken by the frequency of one specific allele at regular time intervals (right). Multiplying this sum by $\tau$ amounts to approximating the integral $\int x_t d_t$ by the histogram shown on the right panel. Thus, $\int x_t dt \simeq \tau \sum x_i = \tau$. This argument can more generally be used to approximate sums of the form $\sum x_i^k \theta_i^l$ by the corresponding integrals, $\frac{1}{\tau} \left( \int x_t^k \theta_t^l dt \right)$, all of which are analytically available.

\newpage

\part*{Appendix}
\section*{Approximation for the fitness function}
Let us define $\R_{t} =\sum_{i  \in K_{t}} x_{i,t} \Rp _{i,t}$.

One can use a Taylor approximation to linearise the fitness function around $\R $.  

\begin{align}
    \overline{\omega_{i,t}} - f(\R_{t} ) &=
    \sum_{j \in K_{t}} x_{j,t} f \left( \dfrac{\Rp _{i,t} + \Rp _{j,t}}{2} \right) - \sum_{j \in K_{t}} x_{j,t} f(\R_{t} ) \\
    &=
    \sum_{j \in K_{t}} x_{j,t}  \left[ f \left( \dfrac{\Rp _{i,t} + \Rp _{j,t}}{2} \right) - f(\R_{t} ) \right] \\
    &\simeq
    \sum_{j \in K_{t}} x_{j,t}  f'(\R_{t} ) \left( \dfrac{\Rp _{i,t} + \Rp _{j,t}}{2} - \R_{t}  \right) \\
    &\simeq
     f'(\R_{t} ) \left( \dfrac{\Rp _{i,t} + \sum_{j \in K_{t}} x_{j,t} \Rp _{j,t}}{2} - \R_{t}  \right) \\
     &\simeq
     f'(\R_{t} ) \left( \dfrac{\Rp _{i,t} - \R_{t} }{2}\right) \\
\end{align}

Consequently $\overline{\omega_{t}} \simeq f(\R_{t} )$. \textbf{Proof},

\begin{align}
    f(\R_{t} ) &= \sum_{i \in K_{t}} x_{i,t} f(\R_{t} ) \\
    &\simeq \sum_{i \in K_{t}} x_{i,t} \left[ \overline{\omega_{i,t}} - f'(\R_{t} ) \left( \dfrac{\Rp _{i,t} - \R_{t} }{2}\right) \right] \\
    &\simeq
    \sum_{i \in K_{t}} x_{i,t} \overline{\omega_{i,t}} - f'(\R_{t} ) \sum_{i \in K_{t}} x_{i,t}  \left( \dfrac{\Rp _{i,t} - \R_{t} }{2}\right) \\
    &\simeq
     \overline{\omega_{t}} - f'(\R_{t} )  \left( \dfrac{\sum_{i \in K_{t}} x_{i,t} \Rp _{i,t} - \R_{t} }{2}\right) \\
    &\simeq
     \overline{\omega_{t}}
\end{align}

And the selection coefficient $s_{i,t}$ for the allele $i$ can be easily computed

\begin{equation}
    s_{i,t} = \dfrac{\overline{\omega_{i,t}} - \overline{\omega_{t}}}{\overline{\omega_{t}}}
    \simeq  \dfrac{f'(\R_{t} )}{f(\R_{t} )} \left[ \dfrac{\Rp _{i,t} - \R_{t} }{2} \right]
\end{equation}

\section*{Analytical relation between frequency and activity in the polymorphic regime.}

\begin{align} 
& \left\{
      \begin{aligned}
          \dfrac{\dd x_{t}}{\dd t} &= \dfrac{f'(\R )}{2 f(\R )} \left( \Rp_{t}  - \R \right) x_{t} \\
        \dfrac{\dd \Rp_{t}}{\dd t} &= 
        - \rho x_{t} \Rp_{t} \\
      \end{aligned}
\right. \\
 \Rightarrow
 & \left\{
      \begin{aligned}
          \dfrac{\dd x_{t}}{\dd \Rp_{t} } &= \dfrac{f'(\R )}{2 \rho f(\R )}\left( \dfrac{\R }{\Rp_{t} } -1 \right) \\
        \dfrac{\dd \Rp_{t} }{\dd t} &= 
        - \rho x_{t} \Rp_{t} \\
      \end{aligned}
    \right. \\
 \Rightarrow
  & \left\{
      \begin{aligned}
         x(\Rp_{t} ) &=\dfrac{f'(\R)}{2 \rho f(\R )} (1- \Rp_{t}  + \R  \operatorname{log}(\Rp_{t} )) + x_{\mathrm{initial}}  \\
        \dfrac{\dd \Rp_{t} }{\dd t} &= 
         \dfrac{f'(\R )}{2 f(\R )} [ \Rp_{t} -1- \R  \operatorname{log}(\Rp_{t} )]\Rp_{t}   - \rho x_{\mathrm{initial}} \Rp_{t} \\
      \end{aligned}
    \right.
\end{align}

\section*{$\tau$, $\D$ and $\V$ in polymorphic regime.}
We make use of the three equations :  
\begin{equation}
\left\{
  \begin{aligned}
         x(\Rp_{t} ) &= \dfrac{f'(\R)}{2 \rho f(\R )} (1- \Rp_{t}  + \R  \operatorname{log}(\Rp_{t} )) \\
         \dfrac{\dd t }{\dd \Rp_{t}} &= \dfrac{1}{-\rho x_{t} \Rp_{t}} \\
        0 &= 1-\Rmin  + \R  \operatorname{log}(\Rmin )  \\
  \end{aligned}
   \right.
\end{equation}

\begin{align}
\tau &= \int_{0}^{\infty } x_{t} \dd t  \\
    &= \int_{1}^{\Rmin } x(\Rp_{t} )   \dfrac{\dd t }{\dd  \Rp_{t}} \dd \Rp_{t}    \\
    &= \int_{1}^{\Rmin } x(\Rp_{t} )   \dfrac{1 }{- \rho x(\Rp_{t} ) \Rp_{t} } \dd \Rp_{t}    \\
    &=  \dfrac{-1 }{\rho }  \int_{1}^{\Rmin }   \dfrac{1 }{\Rp_{t} } \dd \Rp_{t}    \\
    &=  \dfrac{-1 }{\rho } \left[ \operatorname{log}(\Rp_{t}) \right]_{1}^{\Rmin }    \\
    &=  \dfrac{-1 }{\rho } \operatorname{log}(\Rmin )    \\
    &=  \dfrac{1 - \Rmin }{\rho \R } \displaybreak[3] \\ 
\D &= \left( \dfrac{\int_{0}^{\infty } x_{t}^2 \dd  t}{ \tau } \right)^{-1} \\
    &=   \tau \left( \int_{0}^{\infty } x_{t}^2 \dd  t \right)^{-1} \\
    &=   \tau  \left(\int_{1}^{\Rmin } x(\Rp_{t} )^2 \dfrac{\dd t }{\dd  \Rp_{t}} \dd \Rp_{t} \right)^{-1} \\
    &=   \dfrac{ \operatorname{log}(\Rmin ) }{-\rho  }  \left( \int_{1}^{\Rmin } x(\Rp_{t} )^2 \dfrac{1 }{- \rho x(\Rp_{t} ) \Rp_{t} } \dd \Rp_{t} \right)^{-1} \\
    &=   \operatorname{log}(\Rmin ) \left( \int_{1}^{\Rmin } \dfrac{x(\Rp_{t} ) }{ \Rp_{t} } \dd \Rp_{t} \right)^{-1} \\ 
    &=  \operatorname{log}(\Rmin ) \left( \int_{1}^{\Rmin } \dfrac{f'(\R)}{2 \rho f(\R )} \dfrac{ 1- \Rp_{t}  + \R  \operatorname{log}(\Rp_{t} ) }{ \Rp_{t} } \dd \Rp_{t} \right)^{-1} \\ 
    &=  \dfrac{2 \rho f(\R )}{f'(\R)} \operatorname{log}(\Rmin ) \left( \left[  - \Rp_{t} + \operatorname{log}(\Rp_{t}) + \dfrac{\R \operatorname{log}(\Rp_{t})^2}{2}  \right]_{1}^{\Rmin } \right)^{-1} \\ 
	&=  \dfrac{2 \rho f(\R )}{f'(\R)} \operatorname{log}(\Rmin ) \left( 1 - \Rmin + \operatorname{log}(\Rmin) + \dfrac{\R \operatorname{log}(\Rmin)^2}{2} \right)^{-1} \\ 
	&=  \dfrac{2 \rho f(\R )}{f'(\R)} \operatorname{log}(\Rmin ) \left( -\R \operatorname{log}(\Rmin) + \operatorname{log}(\Rmin) + \dfrac{\R \operatorname{log}(\Rmin)^2}{2} \right)^{-1} \\ 
	&=  \dfrac{2 \rho f(\R )}{f'(\R)}  \left(   -\R + 1 + \dfrac{\R \operatorname{log}(\Rmin)}{2}  \right)^{-1} \\ 
	&=  \dfrac{2 \rho f(\R )}{f'(\R)}  \left(  \dfrac{ -2\R + 2 + \Rmin - 1 }{2}  \right)^{-1} \\ 
    &= \dfrac{4 \rho f(\R )}{f'(\R )\left[ 1 + \Rmin  - 2 \R   \right]} \displaybreak[3] \\ 
\V &= \dfrac{\int_{0}^{\infty } x_{t}^2  \Rp_{t} \dd  t}{ \tau }  \\
    &=   \dfrac{1}{\tau} \int_{1}^{\Rmin } x(\Rp_{t} )^2 \Rp_{t} \dfrac{\dd t }{\dd  \Rp_{t}} \dd \Rp_{t}  \\
    &=  \dfrac{ \rho \R }{1 - \Rmin }  \int_{1}^{\Rmin } x(\Rp_{t} )^2 \Rp_{t} \dfrac{1 }{- \rho x(\Rp_{t} ) \Rp_{t} } \dd \Rp_{t}  \\
    &=   \dfrac{ \R }{\Rmin - 1} \int_{1}^{\Rmin } x(\Rp_{t} )  \dd \Rp_{t} \\ 
    &=   \dfrac{ \R }{\Rmin - 1} \int_{1}^{\Rmin } \dfrac{f'(\R)}{2 \rho f(\R )} [ 1- \Rp_{t}  + \R  \operatorname{log}(\Rp_{t} )  ] \dd \Rp_{t} \\ 
    &=  \dfrac{f'(\R)}{2 \rho f(\R )} \dfrac{ \R }{\Rmin - 1} \left[  \Rp_{t} - \dfrac{\Rp_{t}^2}{2} - \Rp_{t} \R + \Rp_{t} \R \operatorname{log}(\Rp_{t} )   \right]_{1}^{\Rmin }  \\ 
	&=  \dfrac{f'(\R)}{2 \rho f(\R )} \dfrac{ \R }{\Rmin - 1} \left(  R - \dfrac{1}{2} + \Rmin - \dfrac{\Rmin^2}{2} - \Rmin \R + \Rmin \R \operatorname{log}(\Rmin )   \right)  \\
	&=  \dfrac{f'(\R)}{2 \rho f(\R )} \dfrac{ \R }{\Rmin - 1} \dfrac{ 2 R - 1 + 2\Rmin - \Rmin^2 - 2 \Rmin \R + 2 \Rmin (\Rmin - 1 )  }{2}  \\
	&=  \dfrac{f'(\R)}{2 \rho f(\R )} \dfrac{ \R }{\Rmin - 1} \left( 2 R - \Rmin - 1  + \Rmin + \Rmin^2 - 2 \Rmin \R \right)  \\
	&=  \dfrac{f'(\R)}{2 \rho f(\R )} \dfrac{ \R }{\Rmin - 1} (\Rmin - 1 )\left(  1 + \Rmin  - 2 \R \right)  \\
	&= \dfrac{f'(\R )}{4 \rho f(\R )}\R \left[ 1 + \Rmin  - 2 \R   \right] 
\end{align}


\end{document}

\section*{Discussion}
